project-p1-first-webpage-master/code/style.css

```diff
@@ -0,0 +1,183 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+body {
+  background-color:#faf3dd;
+  margin: 20px 60px 20px 60px;
+  font-family: 'Architects Daughter', cursive;
+}
+
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 400px;
+  }
+
+header {
+  background-image: url("Images/Page.JPG");
+  background-size: cover; 
+  object-position: bottom;
+  border-color: grey; 
+  border-style: double;
+  text-align: center;
+  height: 700px;
+  padding: 100px; 
+}
+
+h1 {
+  font-family: 'Charmonman', cursive;
+  font-family: 'Eagle Lake', cursive;
+  font-family: 'Sunshiney', cursive;
+  text-align: center;
+  font-size: 80px;
+  margin-bottom: 0px;
+}
+
+h2 {
+  font-family: 'Charmonman', cursive;
+  font-family: 'Eagle Lake', cursive;
+  font-family: 'Sunshiney', cursive;
+  font-size: 40px;
+  text-align: center;
+}
+
+h4 {
+  font-family: 'Sunshiney', cursive;
+  font-size: 20px;
+  text-align: center;
+}
+
+.round-image {
+  border-radius: 50%;
+  display: block; 
+  margin-left: auto;
+  margin-right: auto;   
+}
+
+.flower-box {
+  position: relative; 
+}
+
+.pelat_1 {
+  position: absolute;
+  top: 170px;
+  left: 450px; 
+}
+
+.pelat_2 {
+  position: absolute;
+  top: 350px;
+  left: 500px; 
+}
+
+.pelat_3 {
+  position: absolute;
+  top: 60px;
+  left: 660px; 
+}
+
+.pelat_4 {
+  position: absolute;
+  top: 170px;
+  left: 860px; 
+}
+
+.pelat_5 {
+  position: absolute;
+  top: 360px;
+  left: 820px; 
+}
+
+.about-box {
+  background:white; 
+  border: 8px rgb(27, 18, 18); 
+  text-align: center;
+  padding: 20px 20px; 
+}
+
+.flex-parent {
+  height: 200px; 
+  display: flex; 
+  align-items: center; 
+  justify-content: space-around;
+  border: 2px solid grey; 
+  margin: 50px; 
+  background-color: #065c6f;
+}
+
+.icon {
+  width: 20%;
+  margin: 40px;
+  padding: 10px; 
+  border: 1px solid grey; 
+  background-color: white; 
+  text-align: center;
+}
+
+.grid-container {
+	border: 2px;
+	display: grid; 
+	grid-template-columns: repeat(4, 1fr); 
```

Comment:
Nice with grid! 
Maybe move the text a little bit to the left here and on lines 128-130.